content/posts/glazier.md

```diff
@@ -135,9 +135,9 @@ Per [https://github.com/google/glazier/tree/master/docs/setup#distribution-point
    - https://github.com/google/glazier/blob/master/docs/actions.md#actions. This page describes all of the existing Glazier actions (aka functions). You can also [create your own actions](https://github.com/google/glazier/blob/master/docs/setup/new_actions.md).
    
 
-1. If desired, customize the various Glazier config files in `glazier-starter-kit/glazier-repo` and/or add new files. 
+1. If desired, fork `glazier-starter-kit/glazier-repo` and/or customize as necessary. 
```

Comment:
Great edit. It's clearer than my original and shorter, win win. 

> 1. If desired, fork `glazier-starter-kit/glazier-repo` and/or customize as necessary.

Maybe it should read
> 1. If desired, fork `glazier-starter-kit/glazier-repo` and customize as you deem necessary.

Thoughts?

Reply:
LGTM